OpenMage/magento-lts
# Replace old magento placeholder images in frontend and adminhtml

### Description (*)
Replace old placeholder images with magento logo with our official logo

### Manual testing scenarios (*)
frontend:
1. Create product without image
2. Flush Catalog Image Cache
3. Check on frontend new placeholder

backend:
I don't know how to test it, but it's definitely fine.

### Contribution checklist (*)
